platforms/documentation/docs/src/snippets/buildCache/http-build-cache/groovy/settings.gradle

```diff
@@ -3,15 +3,15 @@ rootProject.name = 'http-build-cache'
 // tag::http-build-cache[]
 buildCache {
     remote(HttpBuildCache) {
-        url = 'https://example.com:8123/cache/'
+        url = uri('https://example.com:8123/cache/')
     }
 }
 // end::http-build-cache[]
 
 // tag::allow-untrusted-server[]
 buildCache {
     remote(HttpBuildCache) {
-        url = 'https://example.com:8123/cache/'
+        url = uri('https://example.com:8123/cache/')
```

Comment:
:x: I don't think we should do this. We should make assignment from `String` to `URL` work instead. It's such a frequent thing people want to do. And if you look at the two versions, I would argue that this:

```groovy
url = 'https://example.com:8123/cache/'
```

...is more correct than this:

```groovy
url = uri('https://example.com:8123/cache/')
```

What does the additional boilerplate of `uri()` add here?

- it doesn't seem to reduce ambiguity, as there don't seem to be any, and
- I also don't see how it would allow for better tooling either.